server/database/knex.js

```diff
@@ -5,7 +5,7 @@ const knex = require('knex')({
   client: 'pg',
 //  debug: true,
   connection,
+  searchPath: ['wallets', 'public'],
```

Comment:
Based on the knex docs, this should work to change the search path to the wallets schema, but we still need to specify wallets.[tableName] for some reason.